_rules/important-letter-spacing-wide-enough-fsy8c7.md

```diff
@@ -0,0 +1,301 @@
+---
+id: fsy8c7
+name: Important letter spacing in style attributes is wide enough
+rule_type: atomic
+description: |
+  This rule checks that the `style` attribute is not used to prevent adjusting `letter-spacing` by using `!important`, except if it's at least 0.12 times the font size.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
```

Comment:
I'm personally not a fan of taking these rules in this direction. As far as I've heard, this hasn't come from any false positives, so the simplest solution could be to just add an assumption. If we did want to fix this, I think the way to do it would be to adjust the expectation to be something like this:

> the [computed]() value of its [word-spacing]() property is at least 0.16 times the [computed]() value of the [font-size]() of the smallest [visible][] text node descendant; or

It's more intuitive (to me) for the rule to continue selecting the element with the style attribute. If you were to literally implement this rule, a single style attribute could generate a dozen issues because some paragraph with no text size changes could have a couple links in them.